ui/00_message/ui/system_log.xml

```diff
@@ -41,99 +41,99 @@
   </message>
   <message>
     <key>syslog_friend_request</key>
-    <text xml:space="preserve">&lt;NAME TARGET&gt;にフレンド申請をしました</text>
+    <text xml:space="preserve">&lt;NAME TARGET&gt; has been requested to be your friend.</text>
```

Comment:
I think sound better like this "You requested friendship to NAME TARGET", Just suggesting use the best one :)